helpers/calculateEndDate.js

```diff
@@ -4,11 +4,13 @@ var moment = require('moment');
 
 module.exports = function(dust) {
 
-  /*
+/*
   * @description Calculate a customer's check-out date based on their check-in date and the amount of nights they have booked
   * @param {string} startDate start date
   * @param {string} nights amount of nights
   * @param {string} [type=days] the measurement of time to add to
+  * @param {string} format - the moment format string for the return
+  * @param {integer} reduce - a number to reduce the nights by. E.g. tickets reduce by one, as standard
   * @example {@_calcEndDate startDate="07/01/2015" nights="3" type="days" /} output "10/01/2015"
   */
```

Comment:
These comments don't really make sense. You don't have 5 params in this function, you have 4. Also, you `@example` isn't correct as your function is called `_calculateEndDate`.